src/bun.js/api/html_rewriter.zig

```diff
@@ -344,7 +340,9 @@ pub const HTMLRewriter = struct {
         rewriter: *LOLHTML.HTMLRewriter,
         context: LOLHTMLContext,
         response: *Response,
-        input: JSC.WebCore.AnyBlob = undefined,
+        bodyValueBufferer: ?JSC.WebCore.BodyValueBufferer = null,
+        tmp_sync_error: ?JSC.JSValue = null,
```

Comment:
this tmp_sync_error is low quality. you can do better (example: a union(enum) instead of 2 optionals and an error) 

but you can do that in a follow-up